py/kubeflow/kubeflow/ci/admission_webhook_tests_runner.py

```diff
@@ -0,0 +1,5 @@
+# This file is only intended for development purposes
+from kubeflow.kubeflow.ci import base_runner
+
+base_runner.main(component_name="admission_webhook_tests",
+                 workflow_name="ad-wh-tests")
```

Comment:
I'll need to check the length, but it might be that the 1 extra character will cause the length to be over them max 63 characters.

Reply:
Here's the error message I had bumped into while also testing this https://github.com/kubeflow/kubeflow/pull/5653#issuecomment-788179056
```
Invalid value: \"kubeflow-kubeflow-presubmit-shared-ui-tests-5653-bd89676-2624-af49\": must be no more than 63 characters"
```

The workflow name above without the `shared-ui-tests`, which is the value we defined in that PR, has a length of 51 chars. So as long as we use at most 12 chars in our names we should be OK

Reply:
The situation for the runner is different than what runs on AWS, but I kept the naming consistent for the sake of sanity.

Reply:
> I kept the naming consistent with the prow_config.yaml for the sake of sanity.

That's good, I also proposed this name to the workflow to keep the names the same everywhere so we will still be consistent

Reply:
@kimwnasptd I just pushed your suggested changes and added some empty files to test if there are any new issues regarding the name length.

Reply:
@kimwnasptd It seems like there were no problems with the test builds after changing the names, so I am resolving this. If this issue does pop up it can be quickly fixed in the future.